incubator/hnc/config/default/manager_image_patch.yaml

```diff
@@ -11,5 +11,5 @@ spec:
       # based on the current value of the IMG envvar. Please try not to check
       # in changes; leave the default value (currently "controller:latest"
       # unless/until we choose something better).
-      - image: controller:latest
+      - image: controller:kind-local
```

Comment:
Please don't check this in (it's not your fault, see #112)

Reply:
done